modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb

```diff
@@ -47,6 +47,29 @@ def refill
         head :no_content
       end
 
+      def refill_prescriptions
+        ids = params[:ids]
+        begin
+          ids.each do |id|
+            client.post_refill_rx(id)
+          end
+        rescue
+          puts "Error refilling prescription: #{e.message}"
```

Comment:
Do you not have to explicitly `rescue => e` in order to access the exception object? I don't think that used to work. I wonder when that changed.

Reply:
As far as I know, this hasn't changed but I'm not sure why it didn't throw an error. I'm surprised my linter didn't catch it either.